src/Endpoint.js

```diff
@@ -38,16 +38,15 @@ class Endpoint<P, T> {
     return this._stateName
   }
 
-  async postFormData (url: string, params: P): Promise<Response> {
-    if (!this.mapParamsToBody) {
-      throw new Error(`The endpoint ${this.stateName} is not able to post form data!`)
+  async fetchOrPost (url: string, formData: ?FormData): Promise<Response> {
```

Comment:
I don't really understand this name as this always does a `fetch` which uses the POST method.

Reply:
Ah oke now I saw the trinary.

In that case this function should be called `getOrPost`. I'd prefere how it was before this as a GET and a POST usually requires different parameters.

Reply:
Renamed. Right now, no different parameters are needed, so I'd leave it as it is. Less duplicated code that way.